src/modules/simulation/gz_bridge/GZBridge.hpp

```diff
@@ -54,13 +55,22 @@
 #include <uORB/topics/vehicle_global_position.h>
 #include <uORB/topics/vehicle_local_position.h>
 
-#include <ignition/msgs.hh>
-#include <ignition/transport.hh>
-#include <ignition/math.hh>
-#include <ignition/msgs/imu.pb.h>
+#include <gz/msgs.hh>
+#include <gz/transport.hh>
+#include <gz/math.hh>
+#include <gz/msgs/imu.pb.h>
+// #include <gz/msgs/fluid_pressure.pb.h>
 
 using namespace time_literals;
 
+static constexpr auto DEFAULT_HOME_ALT_AMSL = 488.0f; // altitude AMSL at Irchel Park, Zurich, Switzerland [m]
+
+// international standard atmosphere (troposphere model - valid up to 11km) see [1]
+static constexpr auto TEMPERATURE_MSL = 288.15f; // temperature at MSL [K] (15 [C])
```

Comment:
Unused? 

```suggestion
```

If we needed them use `lib/geo/geo.h`.

Reply:
I'm including this as it is what @ahcorde had previously so I'm trying to make this so he can test the other side of it (granted ours is currently commented out).